esp/bindings/SOAP/client/soapclient.cpp

```diff
@@ -124,10 +124,7 @@ int CSoapClient::postRequest(const char* contenttype, const char* soapaction, IR
         multipart->serialize(contenttypestr, requeststr);
     }
 
-    if (getEspLogLevel(rpccall.queryContext())>LogNormal)
-    {
-        DBGLOG("Content type: %s", contenttypestr.str());
-    }
+    LOG(getEspLogCategoryForContext(rpccall.queryContext(), LegacyMsgCatMax), "Content type: %s", contenttypestr.str());
```

Comment:
I'm checking my understanding.

Previously this would only log if the requested logging level was high, and then it would log at DebugMsgThreshold.
Now it will always log - either with category LegacyMsgCatMax, or MsgCatEspElevated;

If so I think this will have the side-effect that all messages above the normal detail level will now get a read lock in CLogMsgManager::rejectsCategory.

Reply:
See comment in https://track.hpccsystems.com/browse/HPCC-24691